packages/matchbox/src/components/Picture/Picture.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { layout, margin, compose } from 'styled-system';
+import styled from 'styled-components';
+import { createPropTypes } from '@styled-system/prop-types';
+import PropTypes from 'prop-types';
+import { pick } from '../../helpers/systemProps';
+import { Box } from '../Box';
+
+const system = compose(layout, margin);
+const StyledFigure = styled.figure`
+  ${system};
+  ${({ seeThrough }) => (seeThrough ? 'mix-blend-mode: multiply;' : '')}
+`;
+
+// Picture Component
+const Picture = React.forwardRef(function Picture(props, userRef) {
+  const { children, seeThrough, role, ...rest } = props;
+  const systemProps = pick(rest, system.propNames);
+  return (
+    <StyledFigure
+      as="figure"
+      m="0"
```

Comment:
Does it make sense to set this since we're using `margin` system props too? Could just be a default?

Reply:
Nvm I see what this is doing